styles.css

```diff
@@ -348,3 +354,219 @@ hr {
   border-radius: 10px;
   border: none;
 }
+
+@media (min-width: 768px) {
+  .header {
+    position: fixed;
+    top: 0;
+    left: 0;
+    right: 0;
+    padding: 26px 142px 26px 155px;
+    border: none;
+  }
+
+  nav .menu_icon {
+    display: none;
+  }
+
+  nav .menu {
+    display: flex;
+    list-style: none;
+    gap: 10px;
+    align-items: center;
+  }
+
+  nav .menu a {
+    padding-inline: 12px;
+    font-weight: 500;
+    font-size: 15px;
+    text-decoration: none;
+    color: #344563;
+  }
+
+  .container {
+    margin-top: 76px;
+    padding: 0;
+    background-image: url("./images/Desktop_bg/Header\ bg.svg");
+    background-position: center;
+    border-bottom-left-radius: 150px;
+    background-size: auto;
+  }
+
+  .main_container {
+    margin: 288px auto 255px;
+  }
+
+  .container h1 {
+    font-size: 48px;
+    color: #091e42;
+  }
+
+  .description {
+    font-size: 20px;
+    max-width: 920px;
+  }
+
+  .social_media {
+    font-size: 16px;
+  }
+
+  .social_media_icons {
+    margin-top: 14px;
+  }
+
+  .works_container {
+    display: grid;
+    gap: 142px;
+  }
+
+  .card {
```

Comment:
Very  good work done @UbdaNam , One very small  thing to add is a border and border radius to your work cards as it is in figma template 😊

**your project**
![border radius](https://user-images.githubusercontent.com/59707859/221217366-a4d07955-283c-428f-a7c8-82c2a0fdc39b.PNG)

**figma :**
![work section background figma](https://user-images.githubusercontent.com/59707859/221217480-bca94a3c-9a9e-4c7b-ab06-5a207205a24a.PNG)